common/sgx/tcbinfo.c

```diff
@@ -431,6 +431,28 @@ static oe_result_t _read_tcb_info(
         itr, end, parsed_info->fmspc, sizeof(parsed_info->fmspc)));
     OE_CHECK(_read(',', itr, end));
 
+    {
+        const uint8_t** old_itr = itr;
```

Comment:
I think we need to save and restore *itr. itr never changes; only *itr changes.

Reply:
Very good catch!

My old change happens to work because *itr will only get updated if the _read_property_name_and_colon returns successfully.

I agree that (*itr) should be persisted  instead of itr.

Addressed